engine/runtime/src/main/java/org/enso/interpreter/node/expression/debug/EvalNode.java

```diff
@@ -67,7 +67,7 @@ RootCallTarget parseExpression(LocalScope scope, ModuleScope moduleScope, String
         InlineContext.fromJava(
             localScope, moduleScope, getTailStatus(), context.getCompilerConfig());
     ExpressionNode expr =
-        context.getCompiler().runInline(expression, inlineContext).getOrElse(null);
```

Comment:
This was causing `NullPointerException` if `context.getCompiler().runInline(expression, inlineContext)` returned `Option.empty`. If the same code would be written in Scala, it would not throw NPE because Scala probably converts this `null` to a function returning null.